addons/sale_timesheet/wizard/project_template_create_wizard.xml

```diff
@@ -0,0 +1,16 @@
+<odoo>
+
+    <record id="sale_timesheet_view_form_simplified_template" model="ir.ui.view">
+        <field name="name">sale.timesheet.project.create.wizard.form</field>
```

Comment:
Could you follow the inherit naming guidelines? [Doc](https://www.odoo.com/documentation/master/contributing/development/coding_guidelines.html#inheriting-xml)